src/index.html

```diff
@@ -8,9 +8,47 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="../src/style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="big__block" data-qa="big">
+      <form
+        class="form"
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <label
+          class="label"
+          for="search-input"
+          data-qa="big"
+        >
+          "Try &quot;Los Angeles&quot;
+        </label>
```

Comment:
it is a label, but you use it as placeholder, it is not correct, and you don't need label here at all